go-fuzz/hub.go

```diff
@@ -73,6 +77,102 @@ type Stats struct {
 	restarts uint64
 }
 
+func parseDictTokenLine(tokenLine *[]byte, tokenLineNo int) *[]byte {
```

Comment:
Please move this function to somewhere at the bottom (after use). It's not the most important function of the file to be first.